recipes/bdwgc/all/conanfile.py

```diff
@@ -116,14 +118,17 @@ def _extract_copyright(self):
         return readme_md[index+len(copyright_header):]
 
     def package(self):
-        tools.save(os.path.join(self.package_folder, "licenses", "COPYRIGHT"), self._extract_copyright())
+        save(self, os.path.join(self.package_folder, "licenses", "COPYRIGHT"), self._extract_copyright())
         cmake = self._configure_cmake()
         cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
```

Comment:
The components are not correctly exposed for the new  (or old) generators

Looking quickly https://github.com/ivmai/bdwgc/blob/af1eede4362a53a32b25c4063df6c2ccc8ed489b/CMakeLists.txt#L813
https://github.com/ivmai/bdwgc/blob/af1eede4362a53a32b25c4063df6c2ccc8ed489b/CMakeLists.txt#L824

There are docs https://github.com/conan-io/conan-center-index/blob/master/docs/v2_migration.md#translating-names-information-to-cmake_target_name-cmake_module_target_name-and-cmake_file_name with code examples 🙏

Reply:
I highly appreciate if someone (more proficient in conanfile.py and installation process) advises me the proper fix.
Thank you